GetIntoTeachingApi/Controllers/GetIntoTeaching/CallbacksController.cs

```diff
@@ -86,5 +86,34 @@ public class CallbacksController : ControllerBase
 
             return Ok(new GetIntoTeachingCallback(candidate));
         }
+
+        [HttpPost]
+        [Route("matchback")]
+        [SwaggerOperation(
+           Summary = "Perform a matchback operation to retrieve a pre-populated MailingListAddMember for the candidate.",
+           Description = @"Attempts to matchback against a known candidate and returns a pre-populated GetIntoTeachingCallback if a match is found.",
+           OperationId = "MatchbackMember",
```

Comment:
MatchbackCandidate

Reply:
I tried to use MatchbackCandidate and it was complaining because it's been already used in another controller. So I guess I need to use something else here